volunteering.md

```diff
@@ -0,0 +1,13 @@
+---
+[ ] add more resources to learn from
+[ ] convert doc into markdown and host at /learn
```

Comment:
There is no document here that needs to be converted. this todo is not necessary